client/src/components/ImageUpload.js

```diff
@@ -0,0 +1,20 @@
+import axios from "../api/axios";
```

Comment:
"Use 'axios' from 'api/axios' for server requests. In this scenario, prefer importing directly from 'axios'."